xc7/make/project_xray.cmake

```diff
@@ -49,7 +49,7 @@ endfunction()
 function(PROJECT_XRAY_TILE)
   set(options FUSED_SITES)
   set(oneValueArgs PART TILE)
-  set(multiValueArgs SITE_TYPES)
+  set(multiValueArgs SITE_TYPES EQUIVALENT_TILES PIN_PREFIX)
```

Comment:
Please add a comment explaining how to use "EQUIVALENT_TILES" and "PIN_PREFIX"

Reply:
Done.

Reply:
Thanks,